src/cityreader/cityreader.py

```diff
@@ -16,12 +27,24 @@
 # should not be loaded into a City object.
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
-    return cities
+
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+    with open('cities.csv') as csv_file:
+        read_csv = csv.reader(csv_file, delimiter=',')
+        lineNum = 0
+        for row in read_csv:
```

Comment:
Curious about why you used this variable, doesn't seem like you need it in a for loop.